apps/web/components/chat/message-item.tsx

```
@@ -616,19 +614,78 @@ export const MessageItem = memo(function MessageItem({
+              move(-1)
+            }
+          }}
+        >
+          {currentAttachment && (
+            <div className="space-y-3">
+              <img src={currentAttachment.url} alt={currentAttachment.filename} className="w-full max-h-[75vh] object-contain rounded" />
+              <div className="flex items-center justify-between text-xs" style={{ color: "#b5bac1" }}>
+                <span>{currentAttachment.filename}</span>
+                {imageIndexes.length > 1 && (
+                  <span>
+                    <button className="px-2 py-1 rounded hover:bg-white/10" onClick={() => move(-1)}>← Prev</button>
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)